### DISCUSSION TITLE
Manager empty after upgrade from 1.0.2 to 1.1.0

### DISCUSSION THREAD

**Discussion by AntonOfTheWoods** [longhorn/longhorn#2437] [Q&A] [answered]
I just did a helm upgrade of a health install from 1.0.2 to 1.1.0. In kubectl everything looks good. I have a new and old version of each of my 5 nodes' engine-image-ei*. When I open the manager, it is now completely empty. There are no visible nodes or volumes. Is there something else I need to do?

**Comment by joshimoo** [comment, marked as answer]
@AntonOfTheWoods The UI path has changed in V1.1.1 you need to use an updated deep link.
See: https://github.com/longhorn/longhorn/issues/2118#issuecomment-748202730

**Reply by AntonOfTheWoods** [reply]
Thanks @joshimoo . I know it's hard but that is a really poor support experience. There is an undocumented bug which is not fixed but the ticket was closed anyway, in spite of the fact the issue has been known about for many months. Instead of having a single information point (upgrade notes) your users have to create tickets/ask on forums because the issue is not fixed, in spite of what the closed ticket would suggest. Not ideal.

**Reply by joshimoo** [reply]
cc @yasker @innobead
